sanic/response.py

```diff
@@ -319,9 +319,31 @@ def html(
     )
 
 
+async def validate_file(
+    request_headers: Header, last_modified: Union[datetime, float, int]
+):
+    try:
+        if_modified_since = parsedate_to_datetime(
+            request_headers.get(
+                "If-Modified-Since",
+                request_headers.get("if-modified-since"),
+            )
```

Comment:
Why the redundancy?

Reply:
Because it could be in lower case. Our dict is not case insensitive, am I right?

Reply:
it is this: https://multidict.readthedocs.io/en/stable/multidict.html#cimultidict

Reply:
Ah, that's nice!